stable/ceph-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{template "fullname" .}}
+  labels:
+    app: {{ template "fullname" . }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+spec:
+  replicas: {{default 1 .Values.replicaCount}}
+  template:
+    metadata:
+      labels:
+        app: {{template "fullname" .}}
```

Comment:
Please apply best practices for labels: https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md
Note that you will also have to update label selectors.

See chart examples for reference: https://github.com/kubernetes/helm/tree/master/docs/examples